pkg/client/connector/command.go

```diff
@@ -204,23 +199,18 @@ func (s *service) connectWorker(c context.Context, cr *rpc.ConnectRequest, k8sCo
 	dlog.Infof(c, "Connected to context %s (%s)", cluster.Context, cluster.Server)
 
 	// Phone home with the information about the size of the cluster
-	s.scout <- func() ScoutReport {
-		report := ScoutReport{
-			Action: "connecting_traffic_manager",
-			PersistentMetadata: map[string]interface{}{
-				"cluster_id":        cluster.GetClusterId(c),
-				"mapped_namespaces": len(cr.MappedNamespaces),
-			},
-		}
-		return report
-	}()
+	s.scout.SetMetadatum(c, "cluster_id", cluster.GetClusterId(c))
```

Comment:
nit: Should we also set the mapped_namespaces with `SetMetadatum` instead of adding it to the Report?

Reply:
It used to be a metadatum, so I kept it that way (added as `PersistentMetadata`). Didn't keep the `namespaces` count as metadatum though, since that felt simply wrong.